### PR TITLE
Alt Text Inclusions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing alt text.

### CODE REVIEW

File: iis/configuration/system.ftpServer/security/ipSecurity/add.md [hunk 1]
```diff
@@ -123,11 +123,11 @@ To support FTP publishing for your Web server, you must install the FTP service.
...
 2. In the **Connections** pane, expand the server name, expand **Sites**, and then FTP site or URL for which you want to add IP restrictions.
 3. In the **Home** pane, double-click the **FTP IPv4 Address and Domain Restrictions** feature.  
-    [![](add/_static/image10.png)](add/_static/image9.png)
+    [![Screenshot of the F T P I P v 4 Address and Domain Restrictions feature.](add/_static/image10.png)](add/_static/image9.png)
 4. In the **FTP IPv4 Address and Domain Restrictions** feature, click **Add Allow Entry...** or **Add Deny Entry...** in the **Actions** pane.  
-    [![](add/_static/image12.png)](add/_static/image11.png)
+    [![Screenshot of the F T P I P v 4 Address and Domain Restrictions feature's Actions pane.](add/_static/image12.png)](add/_static/image11.png)
 5. Enter the IP address that you wish to allow or deny, and then click **OK**.  
-    [![](add/_static/image14.png)](add/_static/image13.png)
+    [![Screenshotof the Add Deny Restriction Rule screen with a focus on the Specific I P v 4 address field.](add/_static/image14.png)](add/_static/image13.png)
```
Comment (MeeraDi) [inline iis/configuration/system.ftpServer/security/ipSecurity/add.md:130]: Need a space between "screenshot of"

File: iis/configuration/system.webServer/management/authorization/providers/add.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ The default installation of IIS 7 and later does not include the **Management Se
 2. In **Server Manager**, click the **Manage** menu, and then click **Add Roles and Features**.
 3. In the **Add Roles and Features** wizard, click **Next**. Select the installation type and click **Next**. Select the destination server and click **Next**.
 4. On the **Server Roles** page, expand **Web Server (IIS)**, expand **Management Tools**, and then select **Management Service**. Click **Next**.  
-    [![](add/_static/image2.png)](add/_static/image1.png) .
+    [![Screenshot of Management Tools folder's contained folders with a focus on the Management Service folder.](add/_static/image2.png)](add/_static/image1.png) .
```
Comment (MeeraDi) [inline iis/configuration/system.webServer/management/authorization/providers/add.md:43]: I find this sentence confusing.

File: iis/configuration/system.webServer/management/authorization/providers/add.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ The default installation of IIS 7 and later does not include the **Management Se
 2. In **Control Panel**, click **Programs and Features**, and then click **Turn Windows Features on or off**.
 3. Expand **Internet Information Services**, then **Web Management Tool**.
 4. Select **IIS Management Service**, and then click **OK**.   
-    [![](add/_static/image8.png)](add/_static/image7.png)
+    [![Screenshot of the Management Service screen with the Enable remote connections optioned being checked.](add/_static/image8.png)](add/_static/image7.png)
```
Comment (MeeraDi) [inline iis/configuration/system.webServer/management/authorization/providers/add.md:73]: connections option

File: iis/configuration/system.webServer/security/authentication/windowsAuthentication/index.md [hunk 4]
```diff
@@ -113,7 +113,7 @@ The default installation of IIS 7 and later does not include the Windows authent
 2. In the **Connections** pane, expand the server name, expand **Sites**, and then the site, application, or Web service for which you want to enable Windows authentication.
 3. Scroll to the **Security** section in the **Home** pane, and then double-click **Authentication**.
 4. In the **Authentication** pane, select **Windows Authentication**, and then click **Enable** in the **Actions** pane.   
-    [![](index/_static/image10.png)](index/_static/image9.png)
+    [![Screenshot of the Authentication pane.](index/_static/image10.png)](index/_static/image9.png)
```
Comment (MeeraDi) [inline iis/configuration/system.webServer/security/authentication/windowsAuthentication/index.md:116]: Less than 40 characters.

**Comment by dereklh7** [comment]
@Rick-Anderson @mcleblanc Added missing alt text. Let me know if everything looks good. Hope you two have a great day!

File: iis/configuration/system.ftpServer/security/authentication/denyByFailure.md [hunk 5]
```diff
@@ -74,7 +74,7 @@ To support denying access through authentication on your FTP server, you must in
 2. In the **Connections** pane, select the server.
 3. In the **Home** pane, double-click the **FTP Logon Attempt Restrictions** feature.  
   
-    [![](denyByFailure/_static/image2.png)](denyByFailure/_static/image1.png)
+    [![Screenshot of the Enable F T P Logon Attempt Restrictions, Maximum number of failed login attempts, and time period in seconds fields.](denyByFailure/_static/image2.png)](denyByFailure/_static/image1.png)
```
Comment (Rick-Anderson) [inline iis/configuration/system.ftpServer/security/authentication/denyByFailure.md:77]: @MeeraDi  it's more efficient for everyone to use the suggestions icon to suggest new text. That what the owner can just click "Add to batch" and at the end "Commit".

Comment (Rick-Anderson) [inline iis/configuration/system.ftpServer/security/authentication/denyByFailure.md:77]: image

Comment (Rick-Anderson) [inline iis/configuration/system.ftpServer/security/authentication/denyByFailure.md:77]:
```suggestion
    [![Screenshot of the Enable FTP Logon Attempt Restrictions, Maximum number of failed login attempts, and time period in seconds fields.](denyByFailure/_static/image2.png)](denyByFailure/_static/image1.png)
```

Comment (Rick-Anderson) [reply iis/configuration/system.ftpServer/security/authentication/denyByFailure.md:77]: image

image

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**